javascript/javascript1/week4/voiceAssistant.js

```diff
@@ -0,0 +1,84 @@
+let name = "";
+let toDoList = [];
+
+function getReply(command){
+    // lowercase command and split it into commandWords array
+   const commandWords = command.toLowerCase().split(" ");
+   let response = null;
+   let toDoText = "";
+
+   if (name !== "" && commandWords[0] == "hello"){
+    console.log("you've already introduced yourself") 
+    return 
+   }
+
+   switch(commandWords[0]) {
+      case "hello":
```

Comment:
This is an interesting way to find the correct response!

Reply:
:)